Description: Resolving dependencies with find, and possibly other tasks, report errors
 caused by the above missing flag.
Author: Panayiotis Karabassis <panayk@gmail.com>
Last-Update: 2011-10-06
Forwarded: https://code.google.com/p/jarjar/issues/detail?id=6
--- a/src/main/com/tonicsystems/jarjar/DepFind.java
+++ b/src/main/com/tonicsystems/jarjar/DepFind.java
@@ -62,7 +62,7 @@
                 try {
                   new ClassReader(in).accept(
                       new DepFindVisitor(classes, entry.getSource(), handler),
-                      ClassReader.SKIP_DEBUG);
+                      ClassReader.SKIP_DEBUG | ClassReader.EXPAND_FRAMES);
                 } catch (Exception e) {
                   System.err.println("Error reading " + entry.getName() + ": " + e.getMessage());
                 } finally {
